app/views/application/_javascript.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <%= yield :javascript %>
 
-<%= render "analytics" %>
+<%= render 'analytics' if Rails.env.production? %>
```

Comment:
Yeah it's a bit verbose for sure. I wanted to make sure that A) we weren't sending analytics data in anything but production, and B) not adding dead javascript code if the `GOOGLE_ANALYTICS_ID` wasn't set.